client_search.go

```diff
@@ -76,3 +84,27 @@ func (c clientSearch) IndexID() string {
 func (c clientSearch) Client() *Client {
 	return c.client
 }
+
+func facetFiltersToStr(i interface{}) string {
+	facetsToStr := ""
+	switch v := i.(type) {
+	case []string:
+		for _, slice := range v {
+			stringSlice := slice
+			facetsToStr += fmt.Sprintf("\"%s\",", stringSlice)
```

Comment:
Same as above, use %q, btw if you can have a quote in your string check if the %q escape the quotes, if it's not the case maybe it will be judicious to escape them.

Reply:
Yes it escape all quotes: https://play.golang.org/p/sh-qd4kSPTO

Reply:
Great, didn't know that!